app/controllers/welcome/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="jumbotron">
+  <div class="container">
+    <h1>Vote Local.</h1>
+    <p>
+        Do you want to be more involved in your community?
+        Have you ever felt like your vote didn't matter?
+        This is your first step to get involved in your local political scene.
+        Sign in to see who currently represents you!
```

Comment:
Is this just a placeholder for now?

Reply:
We could definitely use the amazing 'About this Project' section @ztrokey wrote in the README here!